tutorials/classical_simulation/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Oscillations will always be damped.</p><p>I would either delete this markdown cell, or put it below the plot with "We see on the noisy simulation that Rabi oscillations are damped due to the Doppler shift."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>